### DISCUSSION TITLE
Où vont les données de km saisis dans le suivi d'achat

### DISCUSSION THREAD

**Discussion by Greta712** [betagouv/ma-cantine#1670] [Question / réponse]
Une utilisatrice souhaite  savoir où vont  les infos dans le suivi d’achat  (par ex :lorsque qu'elle coche la caractéristique local  avec les km etc..) elles n’apparaissent pas dans l’export Excel.

**Comment by JenniferStephan** [comment]
c'est plutôt une info que l'on garde nous pour connaître l'usage du mot "local" qui a plusieurs défs.£ l'utilisateur voulait l'avoir sur son export ?

**Reply by Greta712** [reply]
oui exactement, elle souhaite les avoir.

**Comment by valeriemerle** [comment]
peut servir aussi pour les calculs impact env?
